extensions/notebook/src/jupyter/serverInstance.ts

```diff
@@ -25,6 +25,26 @@ const defaultPort = 8888;
 type MessageListener = (data: string | Buffer) => void;
 type ErrorListener = (err: any) => void;
 
+/**
+ * Helper function to enable testing
```

Comment:
This function isn't just used for testing. Please update to be descriptive of what the function is actually doing.

Reply:
Gotcha. Would this be descriptive enough?
* Helper function ensures server instance process stops and enable testing